draft-ietf-ppm-dap.md

```diff
@@ -2625,17 +2626,24 @@ reveal additional information about the honest measurements, leading to a
 privacy violation; or the result may have some property that is desirable to the
 adversary ("stats poisoning").
 
+Depending on the deployment and threat model, there are different ways to address
+Sybil attacks, including, though not limited to:
+
+1. Authenticating clients, as described in {{client-auth}};
+1. Removing client-specific metadata on individual reports, such as through the
+   use of anonymizing proxies in the upload flow, as described in {{anon-proxy}}; or
+1. Applying differential privacy protections, as described in {{dp}}.
+
 ### Client authentication {#client-auth}
 
 In settings where it is practical for each Client to have an identity
 provisioned (e.g., a user logged into a backend service or a hardware device
-programmed with an identity), Client authentication is a highly effective way
-for the Aggregators (or an authenticating proxy deployed between clients and the
-Aggregators; see {{anon-proxy}}) to ensure that all reports come from authentic
-Clients and to enforce policy on things like upload rates. Note that because the
-Helper never handles messages directly from the Clients, reports would have to
-use an extension ({{upload-extensions}}) to convey authentication information to
-the Helper.
+programmed with an identity), Client authentication can help Aggregators (or an
+authenticating proxy deployed between clients and the Aggregators; see
+{{anon-proxy}}) ensure that all reports come from authentic Clients. Note that
+because the Helper never handles messages directly from the Clients, reports
+would have to use an extension ({{upload-extensions}}) to convey authentication
+information to the Helper.
```

Comment:
We should link to https://github.com/cpriebe/draft-priebe-ppm-dap-reportauth if it ever makes it to the Datatracker.